.gitignore

```diff
@@ -23,3 +23,7 @@ dist/
 # macOS
 #
 .DS_Store
+
+# ide files
+.vscode/
+.idea/
```

Comment:
There is also a `.idea` in the project. It is not problem. It has it's own .gitignore file, so this in not needed.